packages/eslint-plugin/tests/rules/prefer-standalone/cases.ts

```diff
@@ -0,0 +1,410 @@
+import { convertAnnotatedSourceToFailureCase } from '@angular-eslint/utils';
+import type { MessageIds } from '../../../src/rules/prefer-standalone';
+
+const messageId: MessageIds = 'preferStandalone';
+
+export const valid = [
```

Comment:
The `valid` & `invalid` test cases here are the same as for `prefer-standalone-component` + new cases for `@Pipe` and `@Directive` (taken from prior commits).